src/map/next_js/utils.ts

```
@@ -0,0 +1,124 @@
+export const resolveNodeValue = (node: Node, scope: Scope): any => {
+    if (!node) return null;
+
+    switch (node.type) {
+        case "StringLiteral":
+        case "NumericLiteral":
+        case "BooleanLiteral":
+            return node.value;
+        case "NullLiteral":
+            return null;
+        case "TemplateLiteral":
+            let result = "";
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)